.github/workflows/validate.yaml

```diff
@@ -22,9 +22,9 @@ jobs:
       GOPATH: ${{ github.workspace }}
     steps:
       - name: Setup Golang
-        uses: actions/setup-go@v4
+        uses: actions/setup-go@v2
         with:
-          go-version: 1.19.x
+          go-version: 1.18.x
```

Comment:
Per Kenjiros' comment about Serving repo. It's probably OK to have this actions run on 1.19.

Reply:
I imagine that could increase the confusion a bit more :) Now we use the new version for builds images, everything else stays at 1.18.